File: pset6/mariomore/mario.py
from cs50 import get_int


def main():
    # Taking input Height:
    pyramid = 0
    height = get_int("Enter Height: ")
    while (height <= 0 or height > 8):
        height = get_int("Enter Height: ")

    # Drawing pyramids
    for i in range(height):  # Nested loop
        for j in range(height):
            if (j >= height - 1 - i):  # Conditions
                print("#", end="")
            else:
                print(" ", end="")
        print("  ", end="")
        for n in range(height):
            if (n <= pyramid):        # Conditions
                print("#", end="")
        print()
        pyramid += 1


if __name__ == "__main__":
    main()

File: pset1/lessmario/mario.c
#include <stdio.h>
#include <cs50.h>

int main()
{
    int height = 0;
    int pyramid = 0;
    //Taking input Height:
    do
    {
        height = get_int("Enter Height: ");
    }
    while (height <= 0 || height > 8);

    //Drawing pyramids
    for (int i = 0 ; i < height ; i++)   //Nested loop
    {
        for (int j = 0 ; j < height ; j++)
        {
            if (!(j >= height - 1 - i))  //Conditions
            {
                printf(" ");
            }
        }
        //Nested loop
            for (int n = 0 ; n < height ; n++)
            {
                if (n <= pyramid)   //conditions
                {
                    printf("#");
                }
            }
            printf(y);
        pyramid++;
    }

}
